### PR TITLE
Added EmployeeAssets api

### PR DESCRIPTION
**Pull request by vishakhatak**

### CODE REVIEW

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs [hunk 1]
```diff
@@ -16,6 +16,8 @@ public interface IAssetAssignmentController
 
         Task<IHttpActionResult> GetAssetAssignmentByIdAsync(int id);
```
Comment (sakshiITT) [inline OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs]: Method name should be more meaningful. Adding async should not be required explicitly. Is there any reason behind it?

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs [hunk 2]
```diff
@@ -16,6 +16,8 @@ public interface IAssetAssignmentController
...
 
+        Task<IHttpActionResult> GetAssetAssignmentsByEmployeeIdAsync(int id);
```
Comment (sakshiITT) [inline OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs]: parameter name should be meaningful, use employeeId instead.

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs [hunk 3]
```diff
@@ -16,6 +16,8 @@ public interface IAssetAssignmentController
...
+
         Task<IHttpActionResult> CreateNewAssetAssignmentAsync([FromBody]Asset_Assignment newAsset);
```
Comment (sakshiITT) [inline OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs]: Rename it to CreateAssetAssignment instead. What new represent here?

File: OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs (hunk 1 above)
Comment (HarishVaishnav707) [reply OnlineAssetManagementTool/OnlineAssetManagementTool.API/Controllers/API/Core/IAssetAssignmentController.cs]: We've added task for that changes, we'll take care in current sprint.
Thanks for your suggestions.

**Pull request merged**